modules/katamari/src/test/ts/atomic/api/struct/AdtTest.ts

```diff
@@ -1,236 +1,239 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { describe, it } from '@ephox/bedrock-client';
+import { assert } from 'chai';
 import fc from 'fast-check';
 import { Adt } from 'ephox/katamari/api/Adt';
 import * as Arr from 'ephox/katamari/api/Arr';
 import * as Fun from 'ephox/katamari/api/Fun';
 
-UnitTest.test('ADT Test', () => {
-  const checkInvalid = (message, f) => {
-    let error = false;
-    try {
-      f();
-    } catch (e) {
-      if (e === 'ADTWHOOPS') {
-        // eslint-disable-next-line no-console
-        console.log('die function incorrectly called');
-      } else {
-        error = true;
+describe('atomic.katamari.api.struct.AdtTest', () => {
+  it('ADT Test', () => {
```

Comment:
This is a very big `it` block. It might be getting out of scope, but it'd be good to split this up into a few different blocks.

Reply:
I've split into two. I think it'd need a bit of rewriting to do it better. Not sure that's worth it, given that we want to ditch Adt.